### PR TITLE
Feature/bug report

### PR DESCRIPTION
**Pull request by keatonkraiger**
added bug report on pwa

### CODE REVIEW

File: PWA/src/views/Timesheet.vue [hunk 1]
```diff
@@ -1,124 +1,182 @@
...
 <script>
+  import { mapFields } from "vuex-map-fields";
+  import { mapMutations } from "vuex";
+
   import FileUploader from "@/components/Forms/FileUploader";
   import ServicesDelivered from "@/components/Forms/ServicesDelivered/ServicesDelivered";
   import Mileage from "@/components/Forms/Mileage/Mileage";
   import { FORM, FILE } from "@/components/Utility/Enums.js";
-
+import mockData from "@/components/Utility/happy_path.json";
```
Comment (iyves) [inline PWA/src/views/Timesheet.vue:123]: This should be deleted

File: PWA/src/views/Timesheet.vue [hunk 2]
```diff
@@ -1,124 +1,182 @@
...
     data: function () {
       return {
         // Expose the imported enums to the html section
         FILE: FILE,
         FORM: FORM,
 
         // The uploaded timesheet, as a .json of parsed values from the backend
-        parsedFileData: null,
-        formChoice: null,
+        parsedFileData: mockData,
```
Comment (iyves) [inline PWA/src/views/Timesheet.vue:138]: This should be set to null (mockData is for debugging only)

File: PWA/src/views/Timesheet.vue [hunk 3]
```diff
@@ -1,124 +1,182 @@
...
 
         // Possible statuses of the uploading the form
-        fileStatus: FILE.INIT,
+        fileStatus: FILE.SUCCESS,
```
Comment (iyves) [inline PWA/src/views/Timesheet.vue:141]: Set this to FILE.INIT -- the file.success is for debugging only

**Comment by ohyonghao** [comment]
This is in release now so we will hold off merging this PR until after beta.

**Pull request merged**